core.go

```diff
@@ -187,6 +188,16 @@ func (c *cb) getGRPCServerOptions() []grpc.ServerOption {
 		grpc.ChainUnaryInterceptor(interceptors.DefaultInterceptors()...),
 		grpc.ChainStreamInterceptor(interceptors.DefaultStreamInterceptors()...),
 	)
+	if c.config.GRPCServerMaxConnectionAgeGraceInSeconds > 0 ||
```

Comment:
Default values of these variables are infinity so is this condition required ?

Reply:
what do you mean ?

Reply:
I have updated the checks